OpenMage/magento-lts
# Remove DISCLAIMER and change Magento -> OpenMage in header

### Description (*)

From discussion #2276

Updated file headers to change Magento -> OpenMage and remove the DISCLAIMER about modifying core files. This removed about 1.9MB in disk space.

### Related Pull Requests


### Fixed Issues (if relevant)


### Manual testing scenarios (*)

Check out this branch and run:

```console
git diff -U0 HEAD~1 | grep '^[+-]' | grep -Ev '^(--- a/|\+\+\+ b/)' | awk '!x[$0]++'
```

That will print all unique changes made in this PR. You should only see comment changes. I will post the output in a follow up comment.

### Questions or comments


### Contribution checklist (*)
